src/speaches/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
this test will pass for sure but it looks like it will not give us confidence for changes in the future

Reply:
Tests added